<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-VC1LEOH" failures="2" tests="6" name="Test" time="0.127" errors="0" timestamp="2023-07-21T06:26:27 PDT">
  <testcase classname="testngproject1.TestNgclass3" name="test1" time="0.013"/>
  <testcase classname="testngproject1.TestNgclass3" name="test2" time="0.002">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="testngproject1.TestNgclass3" name="test3" time="0.022">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testngproject1.TestNgclass3.test3(TestNgclass3.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="testngproject1.TestNgclass4" name="test4" time="0.001"/>
  <testcase classname="testngproject1.TestNgclass4" name="test5" time="0.001">
    <skipped/>
  </testcase> <!-- test5 -->
  <testcase classname="testngproject1.TestNgclass4" name="test6" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testngproject1.TestNgclass4.test6(TestNgclass4.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- test6 -->
</testsuite> <!-- Test -->
